webui/react/src/pages/F_ExpList/glide-table/contextMenu.tsx

```diff
@@ -16,9 +16,7 @@ function useOutsideClickHandler(ref: MutableRefObject<any>, handler: (event: Eve
       if (
         ref.current &&
         !ref.current.contains(event.target) &&
-        !(event.target ? (event.target as Element) : null)?.classList?.contains(
-          'ant-dropdown-menu-title-content',
-        )
+        !(event.srcElement as Element).className?.includes('ant-dropdown')
```

Comment:
`srcElement` is marked as  deprecated. Can we change it to something else?

Reply:
ok it looks like I should continue using `event.target as Element` - this makes the change small, but I continue to see positive results